### ISSUE TITLE
Menu Idea

### ISSUE THREAD

**Issue by MrDowntempo** [richrd/suplemon#169]
The bottom lines of suplemon are clearly inspired by nano.  In nano those 2 lines are neat and tidy since it's a simple editor with few features. Suplemon's is already getting full and crowded, and I imagine it'll only get worse as suplemon gets more features. I propose getting rid of those 2 lines entirely, it'll make suplemon's interface much cleaner when doing simple edits! Instead, how about a hidden drop down menu like elinks uses. In elinks, the menu is hidden until the user presses ESC, then the menu is visible and can be navigated with the arrow keys. Just like a GUI menu, the commands that have keyboard shortcuts have the shortcuts listed next to the command. I think this would ultimately be a cleaner solution, would keep suplemon clean and newbie friendly, but still give us room to grow.

**Comment by richrd** [comment]
I agree that the key legend at the bottom isn't very pretty. I think it's good to have it for the new users though. The menu idea isn't a bad idea, but I wouldn't want it to be the primary way of using different functions. On the other hand it could be a nice way to find new features etc. The current implementation doesn't support this so I'll have to look into this more when I do refactoring based on #161.

I guess you're looking for something like this? ![Terminal Menu](http://knightwise.com/wp-content/uploads/2013/01/wordgrinder_006.png)

**Comment by MrDowntempo** [comment]
I agree! Keyboard shortcuts I think should be the primary way to interface with the editor for almost all commands.m But a menu like you've shown in wordgrinder would allow a lot more discoverability than the two bottom rows in nano style.
